README.md

```diff
@@ -20,3 +20,15 @@ We use Heroku pipelines for managing production and staging environments.
 Heroku-Postgresql databses include *FREE* scheduled backups, we just need to enable them using:
 
 `heroku pg:backups:schedule DATABASE_URL --at '03:00 Europe/Copenhagen' --app APP_NAME`
+
+### Git Branch Naming Strategy
+
+In order to help differentiate git branches a little easier prefix your branch with a relevant keyword.
+
+| Keyword | Meaning                                             |
+|---------|-----------------------------------------------------|
+| feature | new additions                                       |
+| fix     | fix/adjustment                                      |
+| style   | Style changes                                       |
+| spike   | experiments & exploration                           |
+| meta    | changes to credentials, .gitignore, rake tasks, etc |
```

Comment:
```suggestion
| meta    | changes to credentials, .gitignore, rake tasks, etc |
| chore   | background tasks that must be approved by a dev|
```

Reply:
We should be careful not to add too many options else people wont use them.
While still not a fan of the name I do see it's usefulness 🤔